server/lib/accounts.js

```diff
@@ -91,6 +118,27 @@ Accounts.onCreateUser(function(options, user = {}) {
 		}
 	}
 
+	if (!user.active) {
+		user.emails.some((email) => {
```

Comment:
Why are you using `.some((email) => {` if you are not using the `email` variable and you are also not returning a `true` value to stop the `some`? what's the expected result?